common-functionality.tslint.json

```diff
@@ -46,7 +46,6 @@
       true,
       "allow-fast-null-checks"
     ],
-    "no-unused-variable": true,
```

Comment:
It shouldn't be removed, instead of that you can change the severity of this rule to warning only